hack/make-release-artifacts.sh

```diff
@@ -29,9 +29,9 @@ krew_tar_archive="krew.tar.gz"
 krew_zip_archive="krew.zip"
 
 echo >&2 "Creating ${krew_tar_archive} archive."
-tar czvf "out/${krew_tar_archive}" "${bin_dir}"
+pushd "${bin_dir}"; tar czvf "${SCRIPTDIR}/../out/${krew_tar_archive}" ./*; popd
 echo >&2 "Creating ${krew_zip_archive} archive."
-zip -X -q -r --verbose "out/${krew_zip_archive}" "${bin_dir}"
+zip -X -q -j --verbose "out/${krew_zip_archive}" ${bin_dir}/*
```

Comment:
I think it would be better if we do the above syntax (tar) for consistency instead of adding `-j`